pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -88,134 +88,121 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 		return err
 	}
 
-	go utils.WaitToAddLicenseKeyWatch(controller, k8sClient, log, licenseAPIReady)
+	// Make a helper for determining which namespaces to use based on tenancy mode.
+	helper := utils.NewNamespaceHelper(opts.MultiTenant, render.ManagerNamespace, "")
```

Comment:
This should be `render.IntrusionDetectionNamespace` instead of `render.ManagerNamespace` right (copy-paste error)? Looking at this and the next line I wonder if it would make sense to just make `GetWatchNamespaces` a method of `NamespaceHelper` rather than an entirely separate package and function.

Reply:
tbh, I feel like I originally had the GetWatchNamespaces on the helper and then changed it to be a separate function for some reason... I can't recall why though.